test/e2e/specs/plugins/plugins__install-remove.ts

```diff
@@ -40,7 +40,7 @@ describe( DataHelper.createSuiteTitle( 'Jetpack: Plugin' ), function () {
 
 		page = await browser.newPage();
 
-		const testAccount = new TestAccount( 'jetpackRemoteSiteUser' );
+		const testAccount = new TestAccount( 'atomicUser' );
```

Comment:
**question:** Is this change intended to be part of this PR? I see that data of `jetpackRemoteSiteUser` are still used within the test.

Reply:
No. Not sure how that made it in this change when I selectively picked files to include.

Reply:
Pushed revision that removes the change. Thanks for noticing!